spec/integration/inventory_spec.rb

```diff
@@ -25,24 +25,45 @@ module Hawkular::Inventory::RSpec
     end
   end
 
-  describe 'Inventory', vcr: { decode_compressed_response: true } do
+  describe 'Inventory' do
     before(:all) do
       @creds = {
         username: 'jdoe',
         password: 'password'
       }
       @client = Hawkular::Inventory::InventoryClient.create(credentials: @creds)
-      @state = {
-        hostname: 'localhost.localdomain',
-        feed: nil
-      }
+      options = { decode_compressed_response: true }
+      options[:record] = :all if ENV['VCR_UPDATE'] == '1'
+      VCR.use_cassette('Inventory/Helpers/get_feeds', options) do
+        feeds = @client.list_feeds
+        @state = {
+          feed_uuid: feeds[0]
+        }
+      end
+    end
+
+    after(:all) do
+      require 'fileutils'
+      FileUtils.rm_rf "#{VCR.configuration.cassette_library_dir}/Inventory/tmp"
+    end
+
+    let(:cassette_name) do |e|
+      description = e.description
+      description
```

Comment:
```bash
Failures:

  1) Inventory Should list feeds
     Failure/Error: record('Inventory', @state, cassette_name, example: example)
     
     ArgumentError:
       no receiver given
     # ./spec/integration/inventory_spec.rb:57:in `block (2 levels) in <module:RSpec>'
...
25 failures
```

The `cassette_name` can't be used in the `around` interceptor.

I found less evil to have 

```
      description = example.description
      description
```
...in the code, rather than rubocop the very first consciously ignored offense :)